test/contains.js

```diff
@@ -28,6 +28,36 @@ describe('contains', function() {
     assert.strictEqual(R.contains(new Just([42]), [new Just([42])]), true);
   });
 
+  it('dispatches to `indexOf` method', function() {
```

Comment:
i think this test should be removed. i'm fine with this implementation of `contains`, but this test lifts the skirt too much.